app/src/main/java/com/framgia/englishconversation/screen/timeline/ItemTimelineViewModel.java

```diff
@@ -0,0 +1,57 @@
+package com.framgia.englishconversation.screen.timeline;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+
+import com.framgia.englishconversation.BR;
+import com.framgia.englishconversation.data.model.TimelineModel;
+import com.framgia.englishconversation.data.model.UserModel;
```

Comment:
ERROR: Unused import: com.framgia.englishconversation.data.model.UserModel. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports